src/components/form/Obs.jsx

```diff
@@ -152,11 +160,11 @@ const mapStateToProps = (state, props) => {
   const fullPath = (props.obsGroupContext ? props.obsGroupContext.path + '^' : '') + props.path;
 
   const name = formUtil.obsFieldName(fullPath, concepts);
-
+  const concept = selectors.getConcept(state, getUuid(props.concept));
   return {
     name: name,
     value: props.formContext ? props.formContext.selector(state, name) : null,
-    concept: selectors.getConcept(state, getUuid(props.concept)),
+    concept: { ...concept, ...props.concept },
```

Comment:
@jomadoye cool, it look me a second to understand what is happening here, but I understand now... a concise way to allow users to override via props, nice!